### PR TITLE
Schedule time neutralTertiary -> neutralSecondary

### PR DESCRIPTION
**Pull request by maximus12793**
#### Pull request checklist

- [x] Include a change request file using `$ npm run change`

#### Description of changes

Change the styling for ContextualMenu secondaryText for accessibility

#### Focus areas to test

(optional)

As specified by @peacokle

### CODE REVIEW

**Comment by joschect** [comment]
Adding @betrue-final-final for design review.

File: common/changes/office-ui-fabric-react/schedule_time_secondary_2018-07-02-17-29.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "Schedule time neutralTertiary -> neutralSecondary ",
```
Comment (cliffkoh) [inline common/changes/office-ui-fabric-react/schedule_time_secondary_2018-07-02-17-29.json]: I'm not sure this comment here makes sense... Could you please reword this to better describe the change? E.g. "ContextualMenu: [...]"

**Comment by betrue-final-final** [comment]
You need to pull master to get screener to pass.

**Comment by JasonGore** [comment]
Screener is failing on master...

**Comment by betrue-final-final** [comment]
Yep, it's got this change that removed the placeholder text. I've had to reject this change several times.
image

**Comment by JasonGore** [comment]
I'm just trying to update and merge some of these. :)
